src/services/db/GitFileSystemService.ts

```diff
@@ -1724,6 +1822,33 @@ export default class GitFileSystemService {
     })
   }
 
+  /**
+   * Does not create a new branch if it already exists
```

Comment:
should the comment be "creates a new branch iff it doesn't exist"

Reply:
yes this is a certified derp moment

Reply:
[99bcdd0](https://github.com/isomerpages/isomercms-backend/pull/1172/commits/99bcdd04c4f7725a1ef9a1e04efab3fbc190442a)